public/javascripts/Choropleths/InitializeStreets.js

```diff
@@ -12,7 +12,7 @@ function InitializeStreets(map, params, streetData) {
     let hasUnauditedStreets = params.unauditedStreetColor != null;
 
     function onEachStreetFeature(feature, layer) {
-        let popupContent = `<a href="/audit/street/${feature.properties.street_edge_id}">Click here</a> to explore this street!`;
+        let popupContent = `<a href="/audit/street/${feature.properties.street_edge_id}">${i18next.t('labelmap:click-here')}</a> ${i18next.t('labelmap:explore-street')}`;
```

Comment:
It would probably be best to turn this into a single translation. Would look something like:
```
let popupContent = i18next.t('explore-street-link', { streetId: feature.properties.street_edge_id });
```

Then the translation looks something like:
```
"explore-street-link": "<a href='/audit/street/{{streetId}}'>Click here</a> to explore this street!"
```